TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -1163,6 +1163,103 @@ def makeTreeFromMiniAOD(self,process):
             'trackFilter:trkshitpattern(Tracks_hitPattern)',
         ])
 
+
+    ## ----------------------------------------------------------------------------------------------
+    ## AK15 jets
+    ## ----------------------------------------------------------------------------------------------
+
+    bTagDiscriminators = [
+        'pfJetProbabilityBJetTags',
+        'pfCombinedInclusiveSecondaryVertexV2BJetTags',
+    ]
+    subjetBTagDiscriminators = [
+        'pfJetProbabilityBJetTags',
+        'pfCombinedInclusiveSecondaryVertexV2BJetTags',
+    ]
+    JETCorrLevels = [
```

Comment:
just use the existing `levels` variable already defined in this file and delete this

Reply:
They're different levels though, `levels  = ['L1FastJet','L2Relative','L3Absolute']` and `JETCorrLevels = ['L2Relative', 'L3Absolute', 'L2L3Residual']`. Not sure if this matters?

Reply:
`levels` will automatically have `L2L3Residual` for data, which is the only time it's needed.
`L1FastJet` is a dummy correction for PUPPI, so it doesn't matter.